src/app/containers/RadioSchedule/index.jsx

```diff
@@ -49,17 +49,23 @@ const RadioScheduleContainer = ({ endpoint }) => {
       propSatisfies(time => time < currentTime, 'publishedTimeStart'),
     )(radioScheduleData.schedules);
 
-    console.log(latestProgrammeIndex);
-    
     const schedulesToShow = radioScheduleData.schedules.slice(
       latestProgrammeIndex - 2,
       latestProgrammeIndex + 2,
     );
-    const latestProgram = radioScheduleData.schedules[latestProgrammeIndex];
-
-    console.log('current time', new Date());
 
-    console.log('current time from var', new Date(currentTime));
+    const latestProgram = radioScheduleData.schedules[latestProgrammeIndex];
+    if (
+      getProgramState(
+        currentTime,
+        latestProgram.publishedTimeStart,
+        latestProgram.publishedTimeEnd,
+      ) === 'live'
+    ) {
+      // Display live program card at the beginning of the schedule
+      schedulesToShow.splice(0, 0, latestProgram);
+      schedulesToShow.splice(3, 1);
```

Comment:
Is there ever a case where there's only one on-demand programme in the data, and a live one? If so, this logic probably would cause problems.

Reply:
We should always get ares data with at least 4 radio programs, so there won't be cases with only 1, 2 or 3 programs